projects/example-app/src/app/books/effects/collection.effects.ts

```diff
@@ -1,16 +1,15 @@
 import { Injectable } from '@angular/core';
-import { Database } from '@ngrx/db';
-import { Actions, Effect, ofType } from '@ngrx/effects';
-import { Action } from '@ngrx/store';
-import { defer, Observable, of } from 'rxjs';
-import { catchError, map, mergeMap, switchMap, toArray } from 'rxjs/operators';
-
-import { Book } from '@example-app/books/models/book';
 import {
-  SelectedBookPageActions,
-  CollectionPageActions,
   CollectionApiActions,
+  CollectionPageActions,
+  SelectedBookPageActions,
 } from '@example-app/books/actions';
+import { Book } from '@example-app/books/models/book';
+import { BookStorageService } from '@example-app/core/services/book-storage.service';
+import { Actions, Effect, ofType } from '@ngrx/effects';
```

Comment:
Keep the import order as before, import `rxjs` and `@ngrx` packages before exame-app imports.

Reply:
`VsCode` default format on save by the looks of it :( Ill revert the order change